api/src/glog.py

```diff
@@ -1,26 +1,33 @@
 import logging
+from datetime import datetime
 from typing import TextIO
 
-from pythonjsonlogger import jsonlogger
+from pythonjsonlogger.json import JsonFormatter as pjlJsonFormatter
 
 
+class JsonFormatter(pjlJsonFormatter):
```

Comment:
Our previous logger setup kept throwing errors when trying to log. This was causing problems when trying to debug so I fixed it.